.github/workflows/python_wheels.yml

```diff
@@ -169,12 +169,24 @@ jobs:
           image: wheel_validation:local
           shell: bash
           run: |
-            python${{ inputs.python_version }} -m pytest /tmp/tests/
+            python${{ inputs.python_version }} -m pytest --ignore /tmp/tests/backends /tmp/tests/
             pytest_status=$?
             if [ ! $pytest_status -eq 0 ]; then
               echo "pytest status = " $pytest_status
               exit 1
             fi
+            apt-get update && apt-get install -y git 
+            git clone https://github.com/nvidia/cuda-quantum
+            cd cuda-quantum
+            python${{ inputs.python_version }} -m pip install --user fastapi uvicorn llvmlite 
```

Comment:
This looks like it is because the installation is missing some required files?

Reply:
That is correct, the mock server needs these modules. If I put them in the main Docker files, we'd have to rebuild the CI cache :/